test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,267 @@
+    await utilities.pause(10);
+  });
+
+  step('Run SFDX: Authorize a Dev Hub', async () => {
+    // This is essentially the "SFDX: Authorize a Dev Hub" command, but using the CLI and an auth file instead of the UI.
+    const workbench = await browser.getWorkbench();
+    await utilities.pause(1);
+
+    const authFilePath = path.join(projectFolderPath, 'authFile.json');
+    const terminalView = await utilities.executeCommand(workbench, `sfdx force:org:display -u ${EnvironmentSettings.getInstance().devHubAliasName} --verbose --json > ${authFilePath}`);
+
+    const authFilePathFileExists = fs.existsSync(authFilePath);
+    expect(authFilePathFileExists).toEqual(true);
+
+    await terminalView.executeCommand(`sfdx auth:sfdxurl:store -d -f ${authFilePath}`);
+
+    const terminalText = await utilities.getTerminalViewText(terminalView, 60);
+    expect(terminalText).toContain(`Successfully authorized ${EnvironmentSettings.getInstance().devHubUserName} with org ID`);
+  });
+
+  step('Run SFDX: Set a Default Org', async () => {
+    // This is "SFDX: Set a Default Org", using the button in the status bar.
+    // Could also run the command, "SFDX: Set a Default Org" but this exercises more UI elements.
+
+    // Click on "No default Org Set" (in the bottom bar).
+    const workbench = await browser.getWorkbench();
+    const statusBar = workbench.getStatusBar();
+    const changeDefaultOrgSetItem = await utilities.getStatusBarItemWhichIncludes(statusBar, 'Change Default Org');
+    expect(changeDefaultOrgSetItem).not.toBeUndefined();
+    await changeDefaultOrgSetItem.click();
+    await utilities.pause(1);
+
+    // In the drop down menu that appears, select "vscodeOrg - user_name".
+    await utilities.selectQuickPickItem(prompt, `${EnvironmentSettings.getInstance().devHubAliasName} - ${EnvironmentSettings.getInstance().devHubUserName}`);
+
+    // Need to pause here for the "set a default org" command to finish.
+    await utilities.pause(5);
+
+    // Look for the notification that appears which says, "SFDX: Set a Default Org successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Set a Default Org successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    const expectedOutputWasFound = await utilities.attemptToFindOutputPanelText('Salesforce CLI', `defaultusername  ${EnvironmentSettings.getInstance().devHubAliasName}  true`, 5);
+    expect(expectedOutputWasFound).not.toBeUndefined();
+
+    // Look for "vscodeOrg" in the status bar.
+    const vscodeOrgItem = await statusBar.getItem(`plug  ${EnvironmentSettings.getInstance().devHubAliasName}, Change Default Org`);
+    expect(vscodeOrgItem).not.toBeUndefined();
+  });
+
+  step('Run SFDX: Create a Default Scratch Org', async () => {
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create a Default Scratch Org...', 1);
```

Comment:
I noticed that this is using the command prompt instead of the org picker, but that doesn't feel like the usual flow for this case. Recognizing it might take more time to test, could we use the org picker's command instead?

Reply:
Since we're testing the org picker below, we're good to keep the command here for now. An additional consideration can be for verifying that drop-down menu of the org picker (if we're not testing that with unit tests already)

Reply:
Spoke with Randi and resolved this issue.